Patches/Black Widows/ThingDefs_Misc/Patch_Apparel_Various.xml

```diff
@@ -87,7 +93,13 @@
 			<li Class="PatchOperationReplace">
 				<xpath>Defs/ThingDef[defName = "Apparel_BWMAGISuitHelm"]/statBases/ArmorRating_Sharp</xpath>
 				<value>
-					<ArmorRating_Sharp>0.55</ArmorRating_Sharp>
+					<ArmorRating_Sharp>14.4</ArmorRating_Sharp>
+				</value>
+			</li>
+			<li Class="PatchOperationReplace">
+				<xpath>Defs/ThingDef[defName = "Apparel_BWMAGISuitHelm"]/statBases/ArmorRating_Blunt</xpath>
+				<value>
+					<ArmorRating_Sharp>32.4</ArmorRating_Sharp>
 				</value>
 			</li>
```

Comment:
Both of these are setting ArmorRating_Sharp for the same def, the bottom one should be blunt.